apps/api/src/user/service/user.service.ts

```diff
@@ -51,6 +84,40 @@ export class UserService {
       isActive: dto.isActive,
       isOnboardingFinished: dto.isOnboardingFinished
     }
+
+    if (dto.email) {
+      const userExists =
+        (await this.prisma.user.count({
+          where: {
+            email: dto.email
+          }
+        })) > 0
+
+      if (userExists) {
+        throw new ConflictException('User with this email already exists')
+      }
+
+      const user = await this.prisma.user.findUnique({
+        where: {
+          id: userId
+        },
+        select: {
+          email: true
+        }
+      })
+
+      const otp = await generateOtp(user.email, userId, this.prisma)
```

Comment:
You wont need to send the OTP if the admin is updating the user.

Reply:
Done